specs/datapackages/README.md

```diff
@@ -0,0 +1,130 @@
+
+Most fields map to CKAN core ones. The ones that are not defined are:
+
+* `encoding`
+* `hash`
+* `sources`
+* `license`
+
+This is not as problematic as dataset fields, as we can add arbitrary top level fields to resources.
+
+## Importing Data Packages
+
+### Implementation
+
+At a higher level this seems quite straight-forward: a  `datapackage_import` or similar action which inherits the auth from `package_create` and `package_update` (eg you need to belong to an org by default etc). The action will validate the JSON object, transform it to a CKAN dict and create or update the relevant package. A controller will handle requests from the UI (at `/datapackage/import`).
+In terms of UI there would be a dedicated page for importing Data Packages. This will essentially contain just an upload field to upload the `datapackage.json` or a text file to enter the URL to an accessible one. This page would show validation errors and redirect to the dataset page once it's created.
+
+Perhaps to make it more accessible we can link to this page from the dataset creation form (from the top bar or the left hand column).
+
+### Potential issues
+
+Identifiers: Data Packages are solely identified by the `name` field, which according to the Data Package spec:
+
+> [is a] short url-usable (and preferably human-readable) name of the package. This MUST be lower-case and contain only alphanumeric characters along with ., _ or - characters. It will function as a unique identifier and therefore SHOULD be unique in relation to any registry in which this package will be deposited (and preferably globally unique).
+
+We can not assume that these identifiers will be unique and that they will not clash with existing CKAN datasets. We can append stuff to the name but then we need to consider what happens when we reupload the same Data Package (ie to update the dataset).
+
+**Q:** Do we upload the files described on the Data Package to the CKAN server?
+I would say not by default. As long as the files are publicly accessible the DataPusher can get them into the DataStore.
+
+**Q:** Do we keep a copy of the original `datapackage.json` ?
+I think yes. Trying to store all fields as extras and replicate them back when generating the datapackage back would be too much hassle. Also we would need to keep adding fields as the spec changes and handle arbitrary fields added by publishers. An easier approach might be outputting back what was uploaded, but with the fields that are actually mapped to CKAN (title, description, keywords etc) updated, as these might have been modified in CKAN.
+
```

Comment:
Is there an ETA on when the spec will be 1.0, and no longer a draft? Having the spec shifting under the feet of the developer seems like it will be problematic, and force things like storing an updated datapackage.json unnecessarily?